cmd/main.go

```diff
@@ -270,7 +269,7 @@ func main() {
 	}
 
 	var ignoredOpenshiftVersions = []string{}
-	parseIgnoredOpenshiftVersions(&ignoredOpenshiftVersions, failOnError)
+	versions.ParseIgnoredOpenshiftVersions(&ignoredOpenshiftVersions, Options.IgnoredOpenshiftVersions, failOnError)
```

Comment:
Why do we need to push `failOnError` in the function? Couldn't we just `failOnError` if the function returns error instead?